docs/sources/alerting/alerting-rules/create-grafana-managed-rule.md

```diff
@@ -87,7 +87,7 @@ Configure alerting behavior in the absence of data using information in the foll
 | Ok             | Set alert rule state to `Normal`.                                                                                                         |
 
 | Error or timeout option | Description                                                                                                                                       |
-| ----------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------- |
+| ----------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------- |
 | Alerting                | Set alert rule state to `Alerting`. From Grafana 8.5, the alert rule waits for the entire duration for which the condition is true before firing. |
 | OK                      | Set alert rule state to `Normal`                                                                                                                  |
-| Error                   | Create a new alert `DatasourceError` with the name and UID of the alert rule, and UID of the datasource that returned no data as labels.          |
+| re                      | Error                                                                                                                                             | Create a new alert `DatasourceError` with the name and UID of the alert rule, and UID of the datasource that returned no data as labels. |
```

Comment:
This looks odd to me.

Reply:
Great catch! This is a multiple cursors error :D I'll fix it.